libs/client/ui/components/src/lib/quick-filter/quick-filter-option.ts

```diff
@@ -0,0 +1,5 @@
+export type QuickFilterOption = {
+  key: string | number;
```

Comment:
Was isch da key?

Reply:
Des isch an unique value, den ma setzen sött, dass in da component logic alles richtig gefiltert werden kann. theoretisch künnts passiera, wemma zwei optionen mit selbem text hot werden immer beide optionen beim klick aktiviert.

Reply:
Ah ok. I schätz so bald ma die Preferences Tabelle hon künna ma den die ID vo da Datenbank dafür verwenda

Reply:
Jo ma künnt die ID us da DB denna als key mitgea. Aba da key brucht ma so oder so als unique identifier.